### DISCUSSION TITLE
How did you learn to make a database?

### DISCUSSION THREAD

**Discussion by clearfram3** [isar/isar#120] [Ideas]
I'm really interested in making my own database and contributing here. What did you study (books, videos) to know how to create a database?

**Comment by stevenkissack** [comment]
It's based on the LMDB Key Value Store, you can find the isar-core repo that uses bindings for the Rust library. Maybe that helps you get started!

**Comment by simc** [comment]
There are many good articles online on how databases work. I would propose to start with key-value DBs then understand how indexes work etc 👍
